README.md

```diff
@@ -1,2 +1,28 @@
-# bdo-emerchant-android
-eMerchant App for BDO Bank 
+# BDO-eMerchant-Android
+{eMerchant App for BDO Bank} 
+
+# Getting Started
+Currently, there is no online channel for merchants to register as a BDO Pay merchant. By developing a mobile
+application, we shall enable online application for both existing and potential partners wherein applicants may provide their
+information and submit required documents through the mobile app. Through this same mobile app, the merchant shall get
+an instanteous response to his application status inquiry.
+
+# Prerequisites
+
+- Android SDK v24
+- Latest Android Build Tools
+- Android Support Repository
+
+# Installation
+
+First download the BDO eMerchant by cloning this repository or downloading an archived snapshot. (See the options at the top of the page.)
+
+In Android Studio, use the "Import non-Android Studio project" or "Import Project" option. Next select one of the sample directories that you downloaded from this repository. If prompted for a gradle configuration accept the default settings.
+
+Alternatively use the "gradlew build" command to build the project directly.
+
```

Comment:
## App Permissions
The first time you open the app, it will ask you to give some permissions to the app. The permissions and the reason why we need the is the following:
-	Camera: This permission is compulsory because we need to capture the KYC details of the merchant.